### PR TITLE
Updated PT Links for Winter 2023

### PR DESCRIPTION
**Pull request by ryqndev**
### Description

Reopened up PT sign up links and updated pt section

Page preview: (updated)
Screen Shot 2023-01-09 at 3 35 35 PM

Also commented out schedule from last quarter. Not deleting it just yet in case we want to reuse it

### CODE REVIEW

File: src/app/pages/PT/PT.js [hunk 1]
```diff
@@ -19,8 +19,21 @@ const PT = () => (
...
+
+				<Text className='color gray'>
+					Applications can be found{' '}
+					<Text className='color blue'>
+						<a
+							href='https://forms.gle/2uHLu1FdWZbGyYeW8'
+							target='_blank'
+							rel='noopener noreferrer'
+						>
+							here.
```
Comment (taesungh) [inline src/app/pages/PT/PT.js:34]: Suggestion: I know we are reusing previous content, but would it be possible to reword this to have more meaningful link text (describe the application, not the word "here")?

Comment (ryqndev) [reply src/app/pages/PT/PT.js:34]: Would be down for it. Do you have any suggestions? Nothing comes to mind for me rn

Comment (taesungh) [reply src/app/pages/PT/PT.js:34]: Maybe if we know the deadline, we could say
> To participate in project teams, [submit an application](#) by \<deadline\>

Comment (ryqndev) [reply src/app/pages/PT/PT.js:34]: can check preview for update

File: src/app/App.js [hunk 2]
```diff
@@ -75,11 +75,11 @@ function App() {
 				/>
 				<Route
 					path='ptsignup'
-					element={<Navigate to='//forms.gle/hHqKT4tX6W1n8VHJ9' />}
+					element={<Navigate to='//forms.gle/2uHLu1FdWZbGyYeW8' />}
 				/>
 				<Route
 					path='ptsignups'
-					element={<Navigate to='//forms.gle/hHqKT4tX6W1n8VHJ9' />}
+					element={<Navigate to='//forms.gle/2uHLu1FdWZbGyYeW8' />}
```
Comment (taesungh) [inline src/app/App.js:82]: Nitpick [non-blocking]: seems we are repeating the PT application URL several times. It might be cleaner to declare and export a constant from `PT.js`.

**Review by taesungh — APPROVED** [review]
Thanks for the update, one small suggestion on refactoring.

File: src/app/App.js (hunk 2 above)
Comment (ryqndev) [reply src/app/App.js:82]: Yeah this is a great idea. Might make this an intern task.
